angularjs-portal-home/src/main/webapp/my-app/layout/widget/partials/lol.html

```diff
@@ -0,0 +1,35 @@
+<div class='widget-body'>
+   <div class='row'>
+      <div class='col-xs-6 center' ng-repeat='item in config.links'>
```

Comment:
Hmm, didn't think about that. I'll refactor to account for different numbers of links.